avocado/core/job.py

```diff
@@ -441,24 +441,24 @@ def pre_tests(self):
         self._result_events_dispatcher.map_method('pre_tests', self)
 
     def run_tests(self):
-        mux = getattr(self.args, "mux", None)
-        if mux is None:
-            mux = multiplexer.Mux()
-        if not mux.is_parsed():   # Mux not yet parsed, apply args
+        variant = getattr(self.args, "xxx_variants", None)
```

Comment:
it's intentional, imagine how probable it is to have `--variants` option. It'd be confusing to me to introduce clashing option just to store internal stuff. So I thought about it and `xxx_` was something which looked as a suitable prefix. Anyway I'm fine to add a different prefix, but I don't think we should use `variants`.